connectors/aws/aws-textract/src/main/java/io/camunda/connector/textract/caller/PollingTextractCaller.java

```diff
@@ -0,0 +1,94 @@
+/*
```

Comment:
You can review this PR with proper diffs, when just selecting the second commit, somehow the renaming of the file is not shown properly here, although i used `git mv`